v/src/Lane.scala

```diff
@@ -410,6 +410,8 @@ class Lane(val parameter: LaneParameter) extends Module with SerializableModule[
 
   val VFUNotClear:           Bool = Wire(Bool())
 
+  val slot0EnqueueFire: Bool = Wire(Bool())
```

Comment:
Is this required? @qinjun-li

Reply:
The first reduce data group needs to give `reduceResult` register an initial value, Slot0EnqueueFire is needed to mark whether the incoming group is the first group.